examples/toy-model-stochastic-degradation.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stochastic degradation model\n",
+    "\n",
+    "This example shows how the stochastic degradation model can be used.\n",
+    "This model describes the stochastic process of a single chemical reaction, in which the concentration of a substance degrades over time as particles react.\n",
+    "The substance degrades starting from an initial concentration, n_0, to 0 following a rate constant, k, according to the following model (Erban et al., 2007):\n",
+    "   $$A \\xrightarrow{\\text{k}} \\emptyset$$\n",
+    "\n",
+    "The model is simulated according to the Gillespie stochastic simulation algorithm (Gillespie, 1976)\n",
+    "   1. Sample a random value r from a uniform distribution: r ~ unif(0,1)\n",
+    "   2. Calculate the time ($\\tau$) until the next single reaction as follows (Erban et al., 2007):\n",
+    "   $$ \\tau = \\frac{1}{A(t)k} \\ln{\\big[\\frac{1}{r}\\big]} $$\n",
+    "   3. Update the molecule count at time t + $\\tau$ as: $ A(t + \\tau) = A(t) - 1 $\n",
+    "   4. Return to step (1) until molecule count reaches 0\n",
+    "   "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pints\n",
+    "import pints.toy\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Specify initial concentration, time points at which to record concentration values, and rate constant value (k)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n_0 = 20\n",
+    "model = pints.toy.StochasticDegradationModel(n_0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "times = np.linspace(0, 100, 100)\n",
```

Comment:
Can you merge cell `In[2]` to `In[4]` into one cell? All seems to me doing one thing.